auditbeat/module/audit/file/fileinfo_bsd.go

```diff
@@ -3,43 +3,62 @@
 package file
 
 import (
-	"fmt"
 	"os"
+	"os/user"
+	"strconv"
 	"syscall"
 	"time"
 
+	"github.com/joeshaw/multierror"
 	"github.com/pkg/errors"
 )
 
-func addFileAttributes(e *Event, maxFileSize int64) error {
-	info, err := os.Lstat(e.Path)
+func Stat(path string) (*Metadata, error) {
+	info, err := os.Lstat(path)
 	if err != nil {
-		return errors.Wrap(err, "failed to stat file")
+		return nil, errors.Wrap(err, "failed to stat path")
```

Comment:
Stat/Lstat return a `*os.PathError` that includes the path already so it would be redundant.